src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
```suggestion
public interface AreaCalculator {
```

немного не информативное название для интерфейса, давай переименуем